<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgKeypad</class>
 <widget class="QDialog" name="DlgKeypad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>401</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Create a Grid of Buttons</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelcol">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>COLUMNS x</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelrow">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ROWS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="col">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="row">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="space_ver">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="space_hor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelrow_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vertical</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcol_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>horizontal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mrg_up">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mrg_lf">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelrow_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelcol_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mrg_dn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelcol_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mrg_rt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelrow_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>MARGINS</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>SPACE between BUTTONS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbScan">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vertical</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>buttonBox</zorder>
  <zorder>labelcol</zorder>
  <zorder>labelrow</zorder>
  <zorder>col</zorder>
  <zorder>row</zorder>
  <zorder>space_ver</zorder>
  <zorder>space_hor</zorder>
  <zorder>labelrow_2</zorder>
  <zorder>labelcol_2</zorder>
  <zorder>mrg_up</zorder>
  <zorder>mrg_lf</zorder>
  <zorder>labelrow_3</zorder>
  <zorder>labelcol_3</zorder>
  <zorder>mrg_dn</zorder>
  <zorder>labelcol_4</zorder>
  <zorder>mrg_rt</zorder>
  <zorder>labelrow_4</zorder>
  <zorder>cmbScan</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgKeypad</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgKeypad</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
